app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<section class="park_info">
+  <h1> <%= @park.name %> Amusement Park </h1>
+  <p> Admission Cost: $<%= @park.admission_cost %> </p>
+</section>
+
+<section class="park_mechanics">
+  <h1> All Mechanics Working at <%= @park.name %> </h1>
+  <ul>
+    <% @park.unique_mechanics.each do |mechanic| %>
+      <li> <%= link_to "#{mechanic.name}", "/mechanics/#{mechanic.id}", data: {turbo_method: :get} %> </li> <br>
+    <% end %>
+  </ul>
+</section>
```

Comment:
This is great. Love the amazing use of HTML here. Just so you know, you didn't have to make each mechanic name a link but it's ok that you did. You also don't need `data: { turbo: :get }` for links, they are `get` by default.